data/json/portal_storm_effect_on_condition.json

```diff
@@ -268,7 +268,8 @@
         [ "EOC_PORTAL_LIGHT", { "const": 2 } ],
         [ "EOC_PORTAL_TELEPORT_STUCK_START", { "const": 3 } ],
         [ "EOC_PORTAL_GRASS_CHANGE", { "const": 5 } ],
-        [ "EOC_PORTAL_MESSAGE", { "const": 5 } ]
+        [ "EOC_PORTAL_MESSAGE", { "const": 5 } ],
+        [ "EOC_PORTAL_ARTIFACT_WEAK", { "const": 1 } ]
```

Comment:
The artifact monsters should be spawning significantly less than they are currently listed.  Since you are already at 1 though that means that all the others need to double.